frontend/src/modules/search/components/FiltersPanel.js

```diff
@@ -298,53 +298,60 @@ export class FiltersPanelBase extends React.Component<Props, State> {
                                         </li>
                                     </Localized>
 
-                                    {props.tagsData.map((tag, i) => {
-                                        const selected = props.tags[tag.slug];
-
-                                        let className = tag.slug;
-                                        if (selected) {
-                                            className += ' selected';
-                                        }
-
-                                        return (
-                                            <li
-                                                className={`tag ${className}`}
-                                                key={i}
-                                                onClick={this.createApplySingleFilter(
-                                                    tag.slug,
-                                                    'tags',
-                                                )}
-                                            >
-                                                <span
-                                                    className='status fa'
-                                                    onClick={this.createToggleFilter(
+                                    {this.props.tagsData
+                                        .sort((a: Tag, b: Tag) => {
```

Comment:
From working with you on this I know that the `Tag` types were added in the attempt to work out what was causing Flow errors. I'm not sure if they are needed here now and will defer to @mathjazz on that.

Reply:
Yeah, please remove the `Tag` types - the rule of the thumb is not to use them as long as Flow doesn't complain.

Please also replace `this.props.tagsData` with `props.tagsData`.

There's a lot of indent in here. One day we should refactor and probably move each Filter section into a separate component.

Reply:
Okay, will remove the `Tag` types, replace with `props.tagData`and recommit the following. Thank you for reviewing!